src/coffee/run.coffee

```diff
@@ -103,8 +103,11 @@ module.exports = class
     program
       .command 'import'
       .description 'Import your products from CSV into your SPHERE.IO project.'
-      .option '-c, --csv <file>', 'CSV file containing products to import'
-      .option '-z, --zip <file>', 'ZIP archive containing multiple product files to import'
+      .option '-c, --csv <file>', 'CSV file containing products to import (alias for "in" parameter)'
+      # add alias for csv parameter and "-i" is taken so use "-f" parameter
+      .option '-f, --in <file>', 'File containing products to import'
+      .option '-z, --zip', 'Input file is archived'
+      .option '-x, --xlsx', 'Import from XLSX format'
```

Comment:
Would using `-f, --format <csv or xlsx>` be an idea here? That way there is no chance for conflicting flags.